tests/rewritetest.cpp

```diff
@@ -593,7 +593,7 @@ TEST_P(ParseTreeComparatorTest, CompareContains)
   auto filter = std::make_unique<execplan::ParseTree>(new execplan::SimpleFilter(
       GetParam().filter, execplan::SimpleFilter::ForTestPurposesWithoutColumnsOIDS{}));
 
-  ASSERT_EQ(GetParam().contains, container.contains(filter));
```

Comment:
What is this and how it is related to logging?

Reply:
oh , boy.. This branch was based on the source tree   that was fixed with @mariadb-LeonidFedorov help as the the compilation of develop branch in bootstrapMCS failed.   My bad for including [tests/rewritetest.cpp]